src/Device.ts

```diff
@@ -61,10 +62,10 @@ Network.get = singleton(() => new NetworkNode());
 const logger = Logger.get("Device");
 
 class Device {
-    async start() {
+    async start(networkInterface : string|undefined = undefined) {
         logger.info(`node-matter@${packageJson.version}`);
 
-        const deviceName = "Matter test device";
+        const deviceName = `Matter test device`;
```

Comment:
No need for a pattern here... revert?

Reply:
Is there a linter getting dropped in soon?  That'll help with these kind of issues. :)

Reply:
Yes inter will come as soon as some PRs got in